src/App.js

```diff
@@ -148,20 +172,23 @@ function App() {
             }}>{playState ? "pause" : "play"}</button>
              <ScrubberBar elapsedTime={elapsedTime} posList={listOfStartStop} eventspot={setTimes} onTimeUpdate={loop}/>
           </div>
-          <button id="startTimeButton" onClick={getStartTimeStamp}>Set Start Timestamp</button>
-          <button id="stopTimeButton" onClick={getStopTimeStamp}>Set Stop Timestamp</button>
-          <button id="clearLoopButton" onClick={clearTime}>Clear Loop</button>
+          <button id="startTimeButton" onClick={getStartTimeStamp}>Set Start Timestamp or Press "S" key</button>
```

Comment:
couuld these 2 buttons be combined into one that handles both start and stop times?

Reply:
handle this next pr maybe not really crucial